### PR TITLE
added ahmed profile

### PR DESCRIPTION
**Pull request by AMagd**

### CODE REVIEW

**Review by sojinoh93 — APPROVED** [review]
Welcome Ahmed!

File: profile/ml/members/ahmed.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Ahmed Magd
+Hello PLAIFers! this is Ahmed from the machine learning team!
+
+![메인 이미지](/assets/ml/ahmed.jpg)
+
+ ## About Me
+ My main focus and work at the present and early in my study is around AI and robotics. Currently I'm focusing on creating creative solutions to improve the performance of real life robots in various tasks.
```
Comment (powerstrong) [inline profile/ml/members/ahmed.md:7]: AI and Robot specialist! You are a well-prepared PLAiFer 😄 Welcome Ahmed!

**Review by powerstrong — APPROVED** [review]

**Comment by wlgjs8** [comment]
Congratulations on joining the PLAIF :) Nice to meet you~

**Review by wlgjs8 — APPROVED** [review]

**Review by gunjae-yoon — APPROVED** [review]
Welcome, Ahmed! I'm looking forward to seeing the meaningful impact you’ll make. I believe you will bring a bright future to us and the AI robotics field.

File: profile/ml/ml.md [hunk 2]
```diff
@@ -9,4 +9,5 @@ ML
 - [🐺 이지훈](/profile/ml/members/jihun.md)
```
Comment (gayounghan) [inline profile/ml/ml.md:9]: @powerstrong 파워스트롱님! 이참에 지훈님 프로필을 모션팀으로 옮기는 건 어떨까요? 🔗

**Review by gayounghan — APPROVED** [review]
Welcome! I feel the same way as Gunjae; I also believe you will bring a bright future to us and the AI robotics field, Ahmed.

**Review by jongwooparkk — APPROVED** [review]
welcome to plaif!!

**Pull request merged**